src/services/index.ts

```diff
@@ -214,7 +214,7 @@ export class Service {
   public buildTx(config: SendConfig) {
     return this.api.tx.utility.batch(
       config.map(({ token, balance, dest }) =>
-        this.api.tx.currencies.transfer(dest, token, balance)
+        this.api.tx.currencies.transfer(dest, token, `${balance}000000000000`)
```

Comment:
Isn't it `balance * 1e18` rather than `balance * 1e12` ?

Reply:
If token decimal in chain_spec is not specified, frontend (apps) use 1e12 by default. Not sure how the parallel frontend decides the decimal though.
We need to adjust this one along with the currencyid redesign.